app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:project_id])
+    @theme = @project.challenge.theme
+    @count = @project.contestants.count
+  end
```

Comment:
We typically only want to send a single instance variable to the view, so just having @project would be sufficient. We also want to make sure any activerecord logic, even those as small as counting, are housed in the model, so I'd love to see an instance method in the Project model called contestant_count that encapsulates the contestants.count logic. Then from the view, we'd just need to call @project.contestant_count to get that data.